sdks/python/container/Dockerfile

```diff
@@ -17,18 +17,15 @@
 ###############################################################################
 
 ARG py_version
-FROM python:"${py_version}"-bullseye as beam
-LABEL Author "Apache Beam <dev@beam.apache.org>"
-ARG TARGETOS
-ARG TARGETARCH
 
-COPY target/base_image_requirements.txt /tmp/base_image_requirements.txt
-COPY target/apache-beam.tar.gz /opt/apache/beam/tars/
-COPY target/launcher/${TARGETOS}_${TARGETARCH}/boot target/LICENSE target/NOTICE target/LICENSE.python /opt/apache/beam/
+FROM python:"${py_version}"-bookworm as python-base
```

Comment:
Thanks!

Just curious - any particular reason to use Bookworm instead of Bullseye?

It's a new version, but it becomes a bit of nonstandard. My guess is that we could bump up the baseline to Bookworm separately

Reply:
Mainly, because Bookworm is new and might have less number of security issues. But this can be changed easily to bullseye (similar to the distroless image).

Reply:
FWIW i don't recall issues with linux distro upgrades in the past.

Reply:
Beam uses vanilla debian as base in other SDKs for now. Are we planning to switch those to distroless as well? ihttps://github.com/apache/beam/blob/9fdc59b2c0f9a54457d9cde3117e9f0c104d59f1/sdks/go/container/Dockerfile#L19

Reply:
Yes.